### PR TITLE
Adds T3 Stack explainer

### PR DESCRIPTION
**Pull request by zomars**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> It is a fast growing stack in popularity. and it would be nice to showcase projects using it.

### CODE REVIEW

File: topics/t3-stack/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+aliases: t3, t3-stack
```
Comment (smashwilson) [inline topics/t3-stack/index.md:2]: You can't use the topic as an alias for itself 😇 . This should get CI green:

```suggestion
aliases: t3
```

File: topics/t3-stack/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+created_by: T3 Open Source
+display_name: T3 Stack
+github_url: https://github.com/t3-oss
+short_description: The T3 Stack is a web development stack focused on simplicity, modularity, and full-stack typesafety.
+topic: t3-stack
+url: https://create.t3.gg
+---
+
+The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t3dotgg) focused on **simplicity**, **modularity**, and **full-stack typesafety**. It consists of:
```
Comment (smashwilson) [inline topics/t3-stack/index.md:11]: Let's simplify this visually a bit:

```suggestion
The T3 Stack is a web development stack made by [Theo](https://twitter.com/t3dotgg) focused on simplicity, modularity, and full-stack type safety. It consists of:
```

**Review by smashwilson — CHANGES_REQUESTED** [review]

File: topics/t3-stack/index.md (hunk 1 above)
Comment (zomars) [reply topics/t3-stack/index.md]: Thanks!

**Comment by zomars** [comment]
Ready to review!

**Comment by smashwilson** [comment]
Thanks for contributing! ❤️

**Pull request merged**